### PR TITLE
303-create-word

### PR DESCRIPTION
**Pull request by hieptm96**

### CODE REVIEW

File: app/controllers/words_controller.rb [hunk 1]
```diff
@@ -12,4 +12,32 @@ def index
...
+    else
+      flash.now[:danger] = t("words.create_fail")
+      render :new
+    end
+  end
+
+  private
+  def word_params
+    params.require(:word).permit :national, :category_id,
+    answers_attributes: [:id, :meaning, :status, :_destroy]
```
Comment (trantube94) [inline app/controllers/words_controller.rb]: indent

File: app/views/words/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  <%= f.label :national %>
+  <%= f.text_field :national %>
+  <br>
+  <%= f.fields_for :answers do |builder| %>
+    <%= builder.label :meaning %>
+    <%= builder.check_box :status %>
+    <%= builder.text_field :meaning %><br>
+  <% end %>
+
+  <%= f.submit 'create' %>
```
Comment (trantube94) [inline app/views/words/new.html.erb]: I18n, '->"

File: config/locales/en.yml [hunk 3]
```diff
@@ -54,10 +54,13 @@ en:
...
+    create_fail: "Create failed!"
+    create_success: "Create successfully!"
     label_all: "All"
     label_learn: "Learned"
     label_notlearn: "Not learned"
     radio_all: "all"
     radio_learn: "learned"
     radio_notlearn: "notlearned"
     submit: "Filter"
+
```
Comment (trantube94) [inline config/locales/en.yml]: ???

**Comment by minhth1905** [comment]
ok :+1:

**Pull request merged**